algorithms/kernel/dtrees/forest/regression/inner/df_regression_training_result_v1.cpp

```diff
@@ -0,0 +1,125 @@
+/* file: df_regression_training_result_v1.cpp */
```

Comment:
you should not transfer this file to inner. You are already handling all cases in df_regression_training_result.cpp. You not have new parameters for dependency result allocate.

Reply:
It was fixed